src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/ProjectView.xaml.cs

```diff
@@ -113,6 +153,9 @@ protected override void OnPreviewKeyDown(KeyEventArgs e)
                         _versions.IsDropDownOpen = false;
                         _versions.SelectedIndex = -1;
                         break;
+                    case Key.Space:
+                        e.Handled = true;
```

Comment:
Nice. 

Is your pasting event also trimming out spaces? Maybe I missed it or it's not obvious.
I'd expect pasting `(1.0.0,  2.0.0 )` to be pasted as `(1.0.0,2.0.0)` just the same as if I typed it.